sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveMetastoreCatalogSuite.scala

```diff
@@ -363,4 +363,33 @@ class DataSourceWithHiveMetastoreCatalogSuite
       }
     })
   }
+
+  Seq(
+    "parquet" -> "org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe",
+    "orc" -> "org.apache.hadoop.hive.ql.io.orc.OrcSerde"
+  ).foreach { case (format, serde) =>
+    test("SPARK-28266: convertToLogicalRelation should not interpret `path` property when " +
+      s"reading Hive tables using $format file format") {
+      withTempPath(dir => {
+        val baseDir = s"${dir.getCanonicalFile.toURI.toString}"
+        spark.range(3).selectExpr("id").write.format(format).save(baseDir)
+        withTable("non_partition_table") {
```

Comment:
Can we give the table a more simple name like `foo` or `t`? Or is it relevant that it's not partitioned?

Reply:
Used simple names now. It is somewhat relevant that the table is non-partitioned. For partitioned tables, the location of each partition is used and so these table level properties are ignored.